src/containers/App/App.js

```diff
@@ -38,8 +39,10 @@ class App extends React.Component {
         const {singleClusterMode, clusterName} = this.props;
         return (
             <AsideNavigation>
-                <Content singleClusterMode={singleClusterMode} clusterName={clusterName} />
-                <div id="fullscreen-root"></div>
+                <ErrorBoundary>
```

Comment:
Is it enough only high-level EB? Maybe it will be usefull to use it also deeper?

Reply:
It's first attempt, I'll add more low-level boundaries when it's will be clear where they are needed